src/super_gradients/recipes/imagenet_regnetY.yaml

```diff
@@ -6,21 +6,21 @@
 #  19 days for RegnetY600, 76.18
 #  20 days for RegnetY800, 77.07
 #  NOTE: Training should probably be lower as resources were shared among the above runs.
-
+#
```

Comment:
Don't think this file was supposed to be in this PR...

Reply:
This was part of previous PR that is now merged. Now it is removed fro this PR.